README.md

```diff
@@ -1,11 +1,283 @@
+
+This tutorial sample shows you how to observe just about everything you can observe from OMSDK `Player` objects. As you would suspect, many properties that can be observed, can also be set or manipulated.
+
+##### Current Playback State and Position <a name="t31"></a>
+
+Determining the current state of the `Player` is a key need for apps … most app-level video playback logic starts here. In addition to the play/pause state, also includes the current position. Once you can query for these property values, you can also programmatically modify them.
+
+##### Looping Playback <a name="t32"></a>
+
+If your app has some need to loop a `Player` (one video or many), such as running a kiosk-style interface, for example. This is an easy operation to accomplish with the OMSDK. Look in this example, to see how to determine when playback finishes, and how to reset the video index back to the first video and start it over.
+
+##### LIVE, VOD, or 360°? <a name="t33"></a>
+
+You may need to inspect some more metadata on the video, such as what type of video this is – LIVE, video on demand, or 360°. This tutorial sample shows how to inspect this. You may need to make certain app design or UX decisions accordingly, based on the type of video that’s currently playing.
+
+##### Manually Hooking up Previous or Next Videos <a name="t34"></a>
+
+There are many legitimate app UX circumstance, that can dictate the dynamicness of a video player – meaning, that not every app design will simply be setup to operate off fixed playlists or lists of videos. As such, the Player can be modified on the fly to dynamically handle what video is played when the previous or next buttons are tapped. This example tutorial has sample code that shows you precisely how to do this. However, be judicious with the usage of this behavior, and make sure it matches a natural flow of content for the user.
+
+### Tutorial 4 – Error Handling in the SDK <a name="t4"></a>
+**insert tutorial link here**
+
+This tutorial sample shows you how to handle various different types of errors that can occur when using the OMSDK and how to catch and identify them. How you handle these in your app is up to you. The SDK is designed to either return a valid SDK or Player instance otherwise it returns an error. There is no middle ground. If you don’t get a valid instance, you should look at the error result instead to determine why. This section describes some common issues.
+
+##### SDK Initialization Errors <a name="t41"></a>
+
+For various reasons, the SDK may fail to initialize. The most common reason for this, is you’re trying to use the OMSDK without first having [onboarded your app’s bundle ID](##Onboarding\ your\ Apps\ for\ SDK\ Authentication). In this case, you’ll get an error that looks like something like this:
+```
+{
+  "error": "Not found - com.company.ungregisteredapp"
+} 
+```
+
+##### Player Initialization Errors <a name="t42"></a>
+
+For various reasons, the `Player` may fail to initialize. 
```

Comment:
Not only player - SDK also.

Reply:
YES!  good catch - please adjust title and text accordingly